### PR TITLE
work on the header

### PR DESCRIPTION
**Pull request by tacya9**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,24 +5,46 @@
...
+			<div class="c-logo"></div>
+			<div class="c-main-layout__header-add-btn">
+				<button class="c-btn">+ <span class="u-only-desktop">add movie</span></button>
+			</div>
+			<div class="c-main-layout__header-search-bar">
+				<h1>find your movie</h1>
+				<form>
+					<ul class="c-main-layout__header-form">
+						<li>
+							<input class="c-input-search" type="text" name="movie" placeholder="What do you want to watch?">
```
Comment (github-mix) [inline index.html:28]: 1) This tag should be closed "/>"
`<input ... />`
2) Why do you need the attribute "name"?

File: styles/base.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	font-size: 16px;
+	font-family: 'Montserrat', sans-serif;
+}
+
+body {
+	height: 100%;
+}
+
+h1 {
+	font-size: 38px;
```
Comment (github-mix) [inline styles/base.scss:12]: let's use 'rem' units for font-size. Check all your files

File: styles/base.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	text-transform: uppercase;
+	color: white;
```
Comment (github-mix) [inline styles/base.scss:14]: let's create a separate file with some variables and use them in other styles. Check all your files.
```
$color-white: #ffffff;
$color-gray: ...
...
$font-primary: "Montserrat", sans-serif;
```

File: styles/base.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+html {
+	height: 100%;
+	font-size: 16px;
+	font-family: 'Montserrat', sans-serif;
```
Comment (github-mix) [inline styles/base.scss:4]: let's use " instead of ' for all styles

File: styles/btn.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.c-btn {
+	text-transform: uppercase;
+	color: #f65261;
+	padding: 14px 20px;
+	border: none;
+	font: normal 900 20px/1 'Montserrat';
+	border-radius: 3px;
+	background: rgba(255,255,255,0.2);
```
Comment (github-mix) [inline styles/btn.scss:8]: please use a color variable:
```
rgba($color-white, 0.2);
```

File: styles/font.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+	font-family: 'Montserrat';
+	font-style: normal;
+	font-weight: 600;
+	font-display: block;
+	src: url("../../fonts/Montserrat-Medium.ttf");
+}
+@font-face {
+	font-family: 'Montserrat';
+	font-style: normal;
+	font-weight: 600;
```
Comment (github-mix) [inline styles/font.scss:18]: you have the same value 600 in the line 11

File: styles/input-search.scss [hunk 7]
```diff
@@ -0,0 +1,10 @@
+.c-input-search {
+	height: 56px;
+	display: inline-block;
+	box-sizing: border-box;
+	font: normal 400 18px/1 'Montserrat';
```
Comment (github-mix) [inline styles/input-search.scss:5]: please use a variable for font-family. Check all your files.
`font: normal 400 18px/1 $font-primary;`

File: styles/main-layout.scss [hunk 8]
```diff
@@ -1,23 +1,76 @@
...
-		padding: 0 18px;
+		border-right: 18px solid transparent;
+		border-left: 18px solid transparent;
 	}
 }
+
 .c-main-layout__header {
-	height: 400px;
+	border-bottom: 10px solid #555555;
+	background: url("../../pictures/header-background.png") repeat-x center black; /* добавить black */
```
Comment (github-mix) [inline styles/main-layout.scss:22]: try to write only in English if you need to add some comments

File: styles/btn.scss [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+	&.m-another-btn {
```
Comment (github-mix) [inline styles/btn.scss:10]: name is not very good )
'm-large' would be better

File: styles/main-layout.scss [hunk 10]
```diff
@@ -1,23 +1,76 @@
...
+	height: 396px;
+	padding-top: 15px;
+	box-sizing: border-box;
+
+	.c-logo {
+		position: absolute;
+		top: 3px;
+	}
+
+	.c-main-layout__header-add-btn {
```
Comment (github-mix) [inline styles/main-layout.scss:32]: why do you need to nest this class here?

File: styles/main-layout.scss [hunk 11]
```diff
@@ -1,23 +1,76 @@
...
+	}
 }
-.c-main-layout__main {
 
+.c-main-layout__header-search-bar {
+	max-width: 840px;
+	font-size: 0;
+	padding: 108px 120px;
+
+	.c-main-layout__header-form {
```
Comment (github-mix) [inline styles/main-layout.scss:44]: why do you need to nest this class here?

File: styles/main-layout.scss [hunk 12]
```diff
@@ -1,23 +1,76 @@
...
+		margin-top: 46px;
+	}
 }
+.c-main-layout__header-form {
+	display: flex;
+
+	> li {
+		margin-left: 15px;
```
Comment (github-mix) [inline styles/main-layout.scss:52]: it's good, but try to use the property 'gap' in order to set some space between cells

File: styles/main-layout.scss [hunk 13]
```diff
@@ -1,23 +1,76 @@
...
+		&:first-child {
+			flex-grow: 1;
+			margin-left: 0;
+		}
+	}
+}
+
+.c-main-layout__main {
+	background: #232323;
+	min-height: calc(100vh - 396px - 70px);
```
Comment (github-mix) [inline styles/main-layout.scss:63]: please avoid the function "calc"
And what will happen if the header or footer height is dynamic?
You really know how to do without "calc"

File: styles/logo.scss [hunk 14]
```diff
@@ -0,0 +1,5 @@
+.c-logo {
+	background: url("../../svg/logo.svg");
```
Comment (github-mix) [inline styles/logo.scss:2]: the path is wrong. Your project starts from this folder, it means that all needed files should be located inside this folder:
image

**Review by github-mix — CHANGES_REQUESTED** [review]

File: styles/base.scss (hunk 3 above)
Comment (tacya9) [reply styles/base.scss:14]: we have discussed and decided to use css variables everywhere

File: styles/variables.scss [hunk 15]
```diff
@@ -0,0 +1,9 @@
+:root {
+	--color-white: #ffffff;
+	--color-carnation: #f65261;
+	--color-emperor: #555555;
+	--color-black: #000000;
+	--color-mine-shaft: #232323;
+	--color-tundora: #424242;
+	--font-primary: "Montserrat", sans-serif;
```
Comment (github-mix) [inline styles/variables.scss:8]: please just add small comments in order to separate types of variables, for example:
```
// color variables
--color-white: #ffffff;
--color-carnation: #f65261;
...
// font variables
--font-primary: "Montserrat", sans-serif;
```

**Review by github-mix — CHANGES_REQUESTED** [review]

**Pull request merged**